gmcs/linglib/lexbase.py

```diff
@@ -197,6 +197,13 @@ def has_evidential(self):
                 return True
         return False
 
+    def has_infostr(self):
+        for lrt in list(self.nodes.values()):
```

Comment:
This list() idiom comes from py2to3, and you should generally avoid it. You don’t need it here.

Reply:
OK!

Reply:
I'll go ahead and clean this out everywhere I can find it, then. Should not be hard. (As usual, the reason I used it is because it was all around in the surrounding code :) ).

Reply:
Done.

Reply:
It's unnecessary for iteration (`for x in something.values(): ...`) but it is still often necessary when you want the values as a list.